Doc/library/codecs.rst

```diff
@@ -23,11 +23,11 @@
 This module defines base classes for standard Python codecs (encoders and
 decoders) and provides access to the internal Python codec registry, which
 manages the codec and error handling lookup process. Most standard codecs
-are :term:`text encodings <text encoding>`, which encode text to bytes,
-but there are also codecs provided that encode text to text, and bytes to
-bytes. Custom codecs may encode and decode between arbitrary types, but some
-module features are restricted to use specifically with
-:term:`text encodings <text encoding>`, or with codecs that encode to
+are :term:`text encodings <text encoding>`, which encode text to bytes (and
+the reverse), but there are also codecs provided that encode text to text, and
+bytes to bytes. Custom codecs may encode and decode between arbitrary types,
+but some module features are restricted to be used specifically with
+ :term:`text encodings <text encoding>` or with codecs that encode to
```

Comment:
It looks like the indentation on this line is causing the Travis doc build issues:
```
/home/travis/build/python/cpython/Doc/library/codecs.rst:30:Unexpected indentation.
```

```suggestion
:term:`text encodings <text encoding>` or with codecs that encode to
```

Reply:
Fixed in local repository.